pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["poetry-core>=1.0.0"]
+requires = ["poetry-core==1.9.1"]
```

Comment:
would <=2.0.0 work?
I'd rather not be so strict as exactly 1.9.1 if we don't have to, but this is currently the only use case, so no strong preference